README.md

```diff
@@ -20,10 +20,10 @@ I'm the maintainer of more than 150 open source projects, such as:
 
 ### Skills & Tools 🖱️
 
-When I need to build things, I usually go for ![vue](https://github.com/Androz2091/Androz2091/raw/main/vue.png) **Vue.js** to code my websites and ![node-js](https://github.com/Androz2091/Androz2091/raw/main/node-js.png) **Node.js** with ![typescript](https://github.com/Androz2091/Androz2091/raw/main/typescript.png) **TypeScript** for my bots and APIs. My favorite database is definitely ![postgresql](https://github.com/Androz2091/Androz2091/raw/main/postgresql.png) **PostgreSQL**.![](https://hit.yhype.me/github/profile?user_id=42497995)
```

Comment:
This syntax seems right for me. I mean, it's a synonym of "I usually opt for Vue" right? I just don't like "**usu**ally **use**" that's just ugly xD